### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by dwes7** [BYU-AUVSI/GroundStation#1]
Would be nice

**Comment by dhill2522** [comment]
I totally agree on this one. Having the installation instructions in `README.md` is pretty standard and important for a project like this.
